lte/gateway/c/core/oai/tasks/amf/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+set(NGAP_C_DIR ${PROJECT_BINARY_DIR}/ngap/r16.0)
```

Comment:
is there an `r16.0` directory?

Reply:
it is ASN specific r16 build folder
![image](https://user-images.githubusercontent.com/83060027/127472567-23081f38-b999-47ad-bbbf-d3d6bd7d8298.png)

Reply:
directory name is `r16` not  `r16.0`. I also do not see where `NGAP_C_DIR` is used.

Reply:
review comment addressed, removed r16  line, NGAP_C_DIR is not used